crates/sui-framework/src/lib.rs

```diff
@@ -89,6 +93,46 @@ pub fn legacy_length_cost() -> InternalGas {
     InternalGas::new(98)
 }
 
+/// Given a `path` and a `build_config`, build the package in that path and return the compiled modules as base64.
+/// This is useful for when publishing via JSON
+pub fn build_move_package_to_base64(
```

Comment:
These look a lot like functions that @sblackshear removed in #5506.  Can you take a look at how their call-sites were replaced in that PR?  It's possible that they are useful in this new context but wanted to flag because they were quite recently removed.

Reply:
if those are still in here it's probably just merge / rebase error - i don't use these.